brreg/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brreg/VirksomhetLoeser.kt

```diff
@@ -107,6 +107,7 @@ class VirksomhetLoeser(
                 eventName = behov.event,
                 transaksjonId = transaksjonId,
                 forespoerselId = behov.forespoerselId?.let(UUID::fromString),
+                Key.AAPEN_ID to json[Key.AAPEN_ID],
```

Comment:
Jeg har ikke noe bedre navn - enda... Men kanskje vi kan tenke litt på det, og kanskje bytte navn før evt prodsetting.. åpen er litt *åpent for tolkning*, hvis du skjønner hva jeg mener. Åpenbart gjør du det.

Reply:
Vi kan godt kaste ball. Jeg er _åpen_ for forslag.